src/v3/src/components/Widget/index.tsx

```diff
@@ -102,16 +102,14 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const handleError = (error: unknown) => {
+  const handleError = (transaction: IdxTransaction | undefined, error: unknown) => {
     // TODO: handle error based on types
     // AuthApiError is one of the potential error that can be thrown here
     // We will want to expose development stage errors from auth-js and file jiras against it
     setAuthApiError(error as AuthApiError);
     console.error(error);
     // error event
-    events?.afterError?.({
-      stepName: idxTransaction?.nextStep?.name,
-    }, error);
+    events?.afterError?.(transaction ? getEventContext(transaction) : {}, error);
```

Comment:
Can you explain a bit what we needed to change here?

Reply:
yeah there was a sneaky test failure that I was trying to get to pass and glen told me he had a very similar one.  So i was trying this fix that he had in his PR.  It looks like the afterError event was being triggered at the wrong time, so the test wasnt able to catch the event and caused it to fail.

Reply:
from this pr https://github.com/okta/okta-signin-widget/pull/2859

Reply:
and the update in line 112 was to match expected data for certain tests